pack.ps1

```diff
@@ -35,7 +37,26 @@ function createNuGetPackage([string]$pkgName, [string]$culture)
     Remove-Item -Path $pkgFolderPath -Recurse
 }
 
-function buildNuGetSpec($pkgName, $culture)
+function createNuGetMetaPackage()
+{
+    $pkgName = $pkgNamePrefix + "All"
+    $pkgId = "$pkgName.$pkgVersion"
+
+    echo "Creating '$pkgId.$pkgExtension' .."
+
```

Comment:
Why do you always use 2 dots I stead of 3?

Reply:
Expected :)